pkg/image/validation/kubernetes.go

```diff
@@ -44,19 +46,53 @@ func isKubernetesDefined(k8s *image.Kubernetes) bool {
 	return k8s.Version != ""
 }
 
-func validateNodes(k8s *image.Kubernetes) []FailedValidation {
+func validateNetwork(k8s *image.Kubernetes) []FailedValidation {
 	var failures []FailedValidation
+	var vipAddress = k8s.Network.APIVIP
+
+	if vipAddress == "" {
+		numNodes := len(k8s.Nodes)
+		// For single node clusters apiVIP is optional, test only multi-node ones
+		if numNodes > 1 {
+			failures = append(failures, FailedValidation{
+				UserMessage: "The 'apiVIP' field is required in the 'network' section when defining entries under 'nodes'.",
+			})
+		}
 
-	numNodes := len(k8s.Nodes)
-	if numNodes <= 1 {
-		// Single node cluster, node configurations are not required
 		return failures
 	}
 
-	if k8s.Network.APIVIP == "" {
+	ip, err := netip.ParseAddr(vipAddress)
+	if err != nil {
+		msg := fmt.Sprintf("Invalid cluster API address ('%s')", strings.ToLower(vipAddress))
 		failures = append(failures, FailedValidation{
-			UserMessage: "The 'apiVIP' field is required in the 'network' section when defining entries under 'nodes'.",
+			UserMessage: msg,
 		})
+
+		zap.S().Error(msg)
```

Comment:
Sorry for misleading you here - it's been a while since I took a look at this portion. I only envisioned we log the error value itself in order not to lose it, but in order to do that we can just populate the `Error` field of the `FailedValidation` and can get rid of this log here.